docusaurus/docs/get-started/set-up-development-environment.mdx

```diff
@@ -439,3 +440,32 @@ Update the `scripts` in the `package.json` to use the extended Webpack configura
 -"dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
 +"dev": "webpack -w -c ./webpack.config.ts --env development",
 ```
+## Add a dependency for local development
+
+Grafana’s plugin development environment is designed to simplify the process of creating and testing plugins. Here’s how it operates:
+
+Plugin distribution path:
+
+The plugin’s dist folder (built plugin) is linked to `/var/lib/grafana/plugins/{{pluginId}}` in the Grafana server environment.
+
+Plugin initialization:
+
+When the Grafana server starts, its plugin loader runs a [Load](https://github.com/grafana/grafana/blob/b53f14ca83eeec6b0f624c2dbbaa417d08bbadbf/pkg/plugins/manager/loader/loader.go#L55-L56) function. This scans the `/var/lib/grafana/` plugins directory for plugins, validates `plugin.json` files, checks signatures and manifests, and initializes valid plugins.
+
+Dependency management:
+
+Although some validations occur on the server side, the plugin path itself doesn't include dependency validation. Developers are responsible for ensuring dependencies are correctly managed.
+
+To manage dependencies during local plugin development, you can use one of the following methods:
+
+### Use the `GF_INSTALL_PLUGINS` environment variable
+
+* Add dependencies explicitly to your Grafana instance with `GF_INSTALL_PLUGINS=dependency`.
```

Comment:
an example of this is any plugin we have in the catalog, so

`GF_INSTALL_PLUGINS=grafana-clock-panel`

tells grafana to install that plugin.

Reply:
added this example.